mmedit/datasets/pipelines/augmentation.py

```diff
@@ -895,3 +962,36 @@ def __repr__(self):
         repr_str = self.__class__.__name__
         repr_str += f'(keys={self.keys}, reverse_ratio={self.reverse_ratio})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class MirrorAugment(object):
+    """Mirror augmentation for training with short sequences (e.g. Vimeo-90K)
+
+    Given a sequence with N frames (x1, ..., xN), augment the sequence to
+    (x1, ..., xN, xN, ..., x1).
+    """
+
+    def __init__(self, keys):
+        self.keys = keys
+
+    def __call__(self, results):
+        """Call function.
+
+        Args:
+            results (dict): A dict containing the necessary information and
+                data for augmentation.
+
+        Returns:
+            dict: A dict containing the processed data and information.
+        """
+        for key in self.keys:
+            if isinstance(results[key], list):
+                results[key] = results[key] + results[key][::-1]
```

Comment:
I suggest that an error should be raised if `results[key]` is not an instance of `list`.